### PR TITLE
add: product listing and filters

### PR DESCRIPTION
**Pull request by harshitpaliwal95**
- Add : filters side bar
- Add : product Listing page
Screenshot 2022-02-12 at 10 45 01 PM

### CODE REVIEW

**Review by sumitch29 — APPROVED** [review]
Looks good.

File: home.css [hunk 1]
```diff
@@ -39,6 +39,10 @@
   padding: 2rem 2rem 2rem 0rem;
 }
 
+/* card badge color */
+.card-badge {
+  background-color: black;
```
Comment (shubhmehra) [inline home.css]: can you use color from your root?

Comment (harshitpaliwal95) [reply home.css:44]: will fix it

File: product/product.html [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+                <div class="card-btn-footer">
+                  <button class="btn btn-outline">Add to Cart</button>
+                </div>
+              </div>
+              <div class="card-component">
+                <div class="card-comp-img">
+                  <img
+                    class="card-top-img"
+                    src="../assets/nike-air-zoom.webp"
+                    alt="loading"
```
Comment (shubhmehra) [inline product/product.html:165]: please use different alt text in every image you can do something like loading-image-1, loading-image-2

**Review by shubhmehra — APPROVED** [review]
Just fix alt message overall looks good to me!

File: product/product.html (hunk 2 above)
Comment (harshitpaliwal95) [reply product/product.html:165]: will change it when shift it to react component

**Pull request merged**